server/txSender/factory.go

```diff
@@ -28,9 +28,9 @@ func CreateTxSender(wallet core.CryptoComponentsHolder, cfg TxSenderConfig) (*tx
 		return nil, err
 	}
 
-	nonceHandler, err := nonceHandlerV2.NewNonceTransactionHandlerV2(nonceHandlerV2.ArgsNonceTransactionsHandlerV2{
-		Proxy:            proxy,
-		IntervalToResend: time.Second * time.Duration(cfg.MaxRetriesSecondsWaitNonce),
+	nonceHandler, err := nonceHandlerV3.NewNonceTransactionHandlerV3(nonceHandlerV3.ArgsNonceTransactionsHandlerV3{
+		Proxy:          proxy,
+		IntervalToSend: time.Millisecond * time.Duration(cfg.IntervalToSend),
```

Comment:
Is it ok to use 1 milisecond?

Reply:
Yes, because in `.env` file the interval is in milliseconds